### PR TITLE
006 deploy frontend

### PR DESCRIPTION
**Pull request by jemorak**
Struggled to get it to access the AWS link - Works locally, just not outside the Docker

### CODE REVIEW

**Review by TomBillingtonUK — CHANGES_REQUESTED** [review]
Any questions/things you don't understand please ask, as there is some new stuff in here

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Node Linter
```
[the remark this answers is not in the document]
Comment (jemorak) [reply .github/workflows/linter.yml:1]: That was Jack's fault. Wrong branch :D

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
[the remark this answers is not in the document]
Comment (jemorak) [reply .vscode/launch.json:1]: I think this file was added automatically when we were trying to add Docker

File: .vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
```
[the remark this answers is not in the document]
Comment (jemorak) [reply .vscode/tasks.json:1]: Like I said before, probably  automatically added when we tried to install Docker

File: .env [hunk 4]
```diff
@@ -0,0 +1,2 @@
+IMAGE_NAME="frontend_app"
+API_URL="https://5tmqdqfjni.eu-west-1.awsapprunner.com/api/test"
```
Comment (TomBillingtonUK) [inline .env:2]: It's still here :)

File: src/config.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+
+// export const API_URL = process.env.API_URL
```
Comment (TomBillingtonUK) [inline src/config.ts:2]: remove this code

**Review by TomBillingtonUK — CHANGES_REQUESTED** [review]
Don't forget the merge conflict :)

**Review by TomBillingtonUK — APPROVED** [review]

**Review by JennicaMistry — APPROVED** [review]

**Pull request merged**